docs/i18n/_extract.py

```
@@ -0,0 +1,342 @@
+# ===== 추출 =====
+rows_by_feature: dict[str, list[dict]] = {}
+
+dart_files = sorted(LIB.rglob("*.dart"))
+for fp in dart_files:
+    rel = relpath(fp)
+    if should_skip_file(rel):
+        continue
+    try:
+        content = fp.read_text(encoding="utf-8")
+    except Exception:
+        continue
```

Ruff (0.15.12)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)